features/profile/view/NewHostRequest.tsx

```diff
@@ -86,14 +85,16 @@ export default function NewHostRequest({
   });
 
   useEffect(() => register("hostUserId"));
+  const empty_arrival_date = t("profile:request_form.arrival_date_empty");
```

Comment:
<img width="773" alt="image" src="https://github.com/Couchers-org/web-frontend/assets/66390686/72961aeb-7c82-4d76-b842-5233b4ed678c">
i get this error text when i try without a date, any idea what might be wrong?

Reply:
(`Error: request_form.arrival_date_empty`) in console

Reply:
that's weird... I'm gonna check it (do you have the changes in the file `features/profile/locales/en.json`?)